src/core/renderPolicy.ts

```diff
@@ -35,7 +35,7 @@ module Plottable {
      * it.
      */
     export class Timeout implements RenderPolicy {
-      public _timeoutMsec: number = Utils.DOM.POLYFILL_TIMEOUT_MSEC;
+      private _timeoutMsec: number = Utils.DOM.POLYFILL_TIMEOUT_MSEC;
```

Comment:
Out of scope, but should we delete this class?  Let's not expose things unless we know they are used